src/main/java/no/sikt/nva/BrageMigrationCommand.java

```diff
@@ -660,9 +660,11 @@ public AwsCredentialsProvider getCredentialsProviderAwsSdk() {
         public AWSCredentialsProvider getCredentialsProviderAwsAuth() {
             switch (this) {
                 case EXPERIMENTAL:
+                    return new DefaultAWSCredentialsProviderChain();
                 case SANDBOX:
+                    return getProfileCredentialsProviderAuth("nva-sandbox-brage-migration");
                 case DEVELOP:
-                    return new DefaultAWSCredentialsProviderChain();
+                    return getProfileCredentialsProviderAuth("nva-dev-brage-migration");
```

Comment:
Nice med denne metoden.

Reply:
Nice, but you should add to the the second method as well:

`public AwsCredentialsProvider getCredentialsProviderAwsSdk()`

We ended up using two different auth libraries in brage-migration, so both implementations are needed:
`software.amazon.awssdk.auth.credentials` 
`com.amazonaws.auth`

Reply:
Or you can refactor so we use only one of implementations :)

Reply:
Nice catch.